python/ray/scripts/scripts.py

```diff
@@ -367,7 +367,7 @@ def debug(address):
     required=False,
     type=int,
     help="The amount of memory (in bytes) to start the object store with. "
-    "By default, this is capped at 20GB but can be set higher.",
+    "By default, this is 30% of available system memory capped by the shm size.",
```

Comment:
Do you want to mention the caps like `DEFAULT_OBJECT_STORE_MAX_MEMORY_BYTES` and `DEFAULT_OBJECT_STORE_MAX_MEMORY_BYTES` as well?